src/views/ExploreSynths.vue

```diff
@@ -259,7 +263,8 @@ export default {
   },
   methods: {
     filter(event) {
-      this.filter_string = event.target.value
+      this.filter_string = event.target.value;
+      console.log(this.synths)
```

Comment:
remove console log after you finish using it
```suggestion
```

Reply:
okay